packages/api-explorer/src/index.jsx

```diff
@@ -1,18 +1,48 @@
 const React = require('react');
 const Cookie = require('js-cookie');
 const PropTypes = require('prop-types');
+const cloneDeep = require('lodash.clonedeep');
 const extensions = require('@readme/oas-extensions');
 const OauthContext = require('@readme/variable/contexts/Oauth');
 const SelectedAppContext = require('@readme/variable/contexts/SelectedApp');
 const { cmVariableContext: TutorialVariableContext } = require('@readme/ui/.bundles/es/views');
 
 const ErrorBoundary = require('./ErrorBoundary');
 const Doc = require('./Doc');
+const { TutorialModal } = require('@readme/ui/.bundles/es/ui/compositions');
 
 const getAuth = require('./lib/get-auth');
 
 const supportedHttpMethods = ['connect', 'delete', 'get', 'head', 'options', 'patch', 'post', 'put', 'trace'];
 
+const defaultTutorial = {
```

Comment:
Can this live in the `TutorialModal` instead as a default prop? The explorer having to know the shape of a tutorial here seems out of scope for the explorer.

Reply:
Yeah good call, I forgot we had it defined there (although a v long path). Fixed this.